misc/assets/mail.svg

```diff
@@ -0,0 +1,60 @@
+<svg version="1.1" viewBox="0 0 1000 400" xmlns="http://www.w3.org/2000/svg">
+  <style>
+      text {
+        font-family: sans-serif;
+      }
+      .caption {
+        font-size: 12pt;
+        text-anchor: middle;
+        dominant-baseline: central;
+      }
+      .note {
+        font-size: 9pt;
+        text-anchor: middle;
+        dominant-baseline: ideographic;
+      }
+      .relation {
+        fill: none;
+        stroke: black;
+        stroke-width: 1pt;
+      }
+      .relation.from-many {
+        marker-start: url(#relation-many)
+      }
+      .relation.polymorphic {
```

Comment:
Aky nazov mam zvolit pre relaciu, kde je nezafarbene koliesko?

Reply:
`from-optional` resp. `to-optional` ak je na konci relacie.

Vid nazvy typov relacii [na obrazku na wikipedii](https://en.wikipedia.org/wiki/Object-modeling_technique)

Reply:
:ok: